pallets/vector/src/lib.rs

```diff
@@ -696,10 +680,84 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::call_index(11)]
+		#[pallet::weight(T::WeightInfo::failed_tx_index(0u32))]
```

Comment:
After using the actual weights, we should consider annotating this as a `Mandatory`  dispatch_class